installer/paddleflow-deployment.yaml

```diff
@@ -0,0 +1,958 @@
+        apiVersion: batch.volcano.sh/v1alpha1
+        kind: Job
+        metadata:
+          name: vcJobName
+        spec:
+          minAvailable: 3
+          schedulerName: volcano
+          priorityClassName: normal
+          policies:
+            - event: PodEvicted
+              action: RestartJob
+          plugins:
+            ssh: []
+            env: []
+            svc: []
+          maxRetry: 5
+          queue: default
+          tasks:
+            - replicas: 1
+              name: "ps"
+              ports:
+                - ps-port: 8001
+              template:
+                metadata:
+                  name: web
+                spec:
+                  containers:
+                    - image: nginx
+                      imagePullPolicy: IfNotPresent
+                      name: ps
+                      resources:
+                        requests:
+                          cpu: "1"
+                  restartPolicy: OnFailure
+            - replicas: 2
+              name: "worker"
+              ports:
+                - worker-port: 8001
+              template:
+                metadata:
+                  name: web
+                spec:
+                  containers:
+                    - image: nginx
+                      imagePullPolicy: IfNotPresent
+                      name: worker
+                      resources:
+                        requests:
+                          cpu: "1"
+                  restartPolicy: Never
+              policies:
+                - event: TaskCompleted
+                  action: CompleteJob
+kind: ConfigMap
+metadata:
+  name: paddleflow-server
+  namespace: paddleflow
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: paddleflow-server
+  namespace: paddleflow
+  labels:
+    app: paddleflow-server
+spec:
+  type: NodePort
+  ports:
+    - port: 8999
+      name: listen
+      targetPort: 8999
+      nodePort: 8999
+  selector:
+    app: paddleflow-server
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: paddleflow-server
+  namespace: paddleflow
+  labels:
+    app: paddleflow-server
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: paddleflow-server
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: paddleflow-server
+    spec:
+      containers:
+        - args:
+            - -c
+            - cd /home/paddleflow/server && ./paddleflow  --pfs-service-name=$(KUBE_NODE_NAME)
+              --pfs-service-port=8999 --db-password=${dbPassword}
+          command:
+            - /bin/sh
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          image: paddleflow/paddleflow-server:1.4.1
```

Comment:
The version of paddleflow-server is `1.4.1`, but the version of `paddleflow-csi-plugin` is `1.4.2` ?

Reply:
all of images is fake and meanless, which would be modified. we need push the final release version images.